core/amber/src/main/python/core/python_worker.py

```diff
@@ -20,6 +41,7 @@ def __init__(self, worker_id: str, host: str, output_port: int):
             port=output_port,
             handshake_port=self._network_receiver.proxy_server.get_port_number(),
         )
+        self.original_parent_pid = 1
```

Comment:
The pid means the process id. `PythonWorker` is a thread/runnable. It might be better to store the parent pid of the process somewhere outside PythonWorker.